<?xml version="1.0"?>
<launch>
  <arg name="camera_topic" default="/optris/image_mono"/>
  <arg name="model_name" default="flir_adas_008"/>
  <arg name="models_dir" default="$(find object_detection_tensorflow)/models"/>
  <arg name="path_to_labels" default="$(find object_detection_tensorflow)/data/flir_adas.pbtxt"/>
  <arg name="num_classes" default="4"/>
  <node pkg="object_detection_tensorflow" type="object_detection_tensorflow.py" name="object_detection_tensorflow_node">
    <param name="camera_topic" value="$(arg camera_topic)"/>
    <param name="models_dir" value="$(arg models_dir)"/>
    <param name="model_name" value="$(arg model_name)"/>
    <param name="path_to_labels" value="$(arg path_to_labels)"/>
    <param name="num_classes" value="$(arg num_classes)"/>
  </node>
  <!-- <node pkg="usb_cam" type="usb_cam_node" name="usb_cam_node"/> -->
</launch>
